project_event/models/project_project.py

```diff
@@ -57,20 +57,32 @@ def _compute_event_log_count(self):
 
     @api.multi
     def action_cancel(self):
+        if self.task_state == 'accepted':
```

Comment:
there is no cancel of the calendar.event to cancel resource reservation?

Reply:
if an event is canceled its activities and tasks must also be canceled.
Same thing for activity (if it is canceled, its tasks should be canceled)

Reply:
Yes I m adding this functionality in a moment

Reply:
Done